src/ui/idam-maintenance-ui/src/FamilyHubs.Idams.Maintenance.Data/FamilyHubs.Idams.Maintenance.Data.csproj

```diff
@@ -2,7 +2,8 @@
 
 	<PropertyGroup>
 		<NuGetAuditMode>direct</NuGetAuditMode>
-    <TreatWarningsAsErrors>True</TreatWarningsAsErrors>
+        <TreatWarningsAsErrors>True</TreatWarningsAsErrors>
+        <SonarQubeExclude>true</SonarQubeExclude>
```

Comment:
You mentioned ignored because it's not user facing. Is that the only reason? Sonar is a tool to improve quality of code which in my head would include internal tools

Reply:
Fair point, reverted: [a7e14c3](https://github.com/DFE-Digital/fh-services/pull/302/commits/a7e14c31f8fe57141769157f7128763c5b8d70a9)